models/mentee.js

```diff
@@ -1,27 +1,27 @@
 'use strict';
```

Comment:
I should have cleaned the mentee.js and mentor.js models and example json files up in my previous change. Would you mind deleting them? We have switched to using just users.js